libs/azure-ai/langchain_azure_ai/vectorstores/azure_cosmos_db_no_sql.py

```diff
@@ -77,6 +62,7 @@ def __init__(
         metadata_key: str = "metadata",
         create_container: bool = True,
         full_text_search_enabled: bool = False,
+        table_alias: str = "c",
```

Comment:
We can leave this hardcoded as `c`, as this is internal to the implementation of the vector store, and won't matter to the user what table-alias name is being used.

Reply:
This is not necessarily so.
If the user adds a `where` clause, the fields must reference the same table.
That is the reason I left it explicit.
If somehow you have a where condition with other table name, you can customize it here